cms/templates/js/content-group-details.underscore

```diff
@@ -41,17 +41,18 @@
 <% if (showContentGroupUsages) { %>
   <div class="collection-references wrapper-group-configuration-usages">
     <% if (!_.isEmpty(usage)) { %>
-      <h4 class="intro group-configuration-usage-text"><%= gettext('This content group is used in:') %></h4>
+      <h4 class="intro group-configuration-usage-text"><%- gettext('This content group is used in:') %></h4>
       <ol class="usage group-configuration-usage">
         <% _.each(usage, function(unit) { %>
           <li class="usage-unit group-configuration-usage-unit">
-            <p><a href=<%= unit.url %> ><%= unit.label %></a></p>
+            <p><a href=<%= unit.url %> ><%- unit.label %></a></p>
```

Comment:
@efischer19 Please let me know what you find regarding this issue.  I'm especially interested if the URL has query params separated by `&` which get escaped to `&amp;`.   I was concerned about this in the past, and thought I had proven this out, but again have the same concerns.  Thanks.